hilti/src/rt/tests/to_string.cc

```diff
@@ -32,9 +33,21 @@ TEST_CASE("string") { CHECK_EQ(to_string(std::string("abc")), "\"abc\""); }
 
 TEST_CASE("vector") {
     CHECK_EQ(to_string(std::vector<int8_t>()), "[]");
-    CHECK_EQ(to_string(std::vector{int8_t{1}}), "[1]");
-    CHECK_EQ(to_string(std::vector{int8_t{1}, 2}), "[1, 2]");
-    CHECK_EQ(to_string(std::vector{std::vector{int8_t{1}, 2}}), "[1, 2]");
+    CHECK_EQ(to_string(std::vector<int8_t>({1})), "[1]");
+    CHECK_EQ(to_string(std::vector<int8_t>({1, 2})), "[1, 2]");
+    CHECK_EQ(to_string(std::vector<std::vector<int8_t>>({{1, 2}})), "[[1, 2]]");
```

Comment:
I'm wondering if/why we still need such support for "normal" vectors. They are probably still produced somewhere, but could we aim for using `Vector` everywhere?

Reply:
The only use case I could think of would be for debugging output in internal code involving `std::vector`. I'll remove `to_string` for `std::vector` for now; if a need arises, we could add an implementation back, but somewhere outside of `hilti::rt`.